app/src/main/java/com/github/se/gomeet/ui/mainscreens/SearchModule.kt

```diff
@@ -0,0 +1,213 @@
+                  }
+                  .background(color = backgroundColor, shape = RoundedCornerShape(10.dp))) {
+            Image(
+                painter = painter,
+                contentDescription = "User Icon",
+                modifier = Modifier.size(24.dp).padding(8.dp))
+            Spacer(modifier = Modifier.width(16.dp))
+            Column {
+              Text(
+                  text = "${item.user.firstName} ${item.user.lastName}",
+                  modifier = Modifier.fillMaxWidth().padding(8.dp),
+                  color = Color.Black)
+              Text("@${item.user.username}", color = Color.Gray)
+            }
+          }
+    }
+    is SearchViewModel.SearchableItem.Event -> {
+      val painter: Painter =
+          if (item.event.images.isNotEmpty()) {
+            rememberAsyncImagePainter(
+                ImageRequest.Builder(LocalContext.current)
+                    .data(data = item.event.images[0])
+                    .apply(
+                        block =
+                            fun ImageRequest.Builder.() {
+                              crossfade(true)
+                              placeholder(R.drawable.gomeet_icon)
+                            })
+                    .build())
+          } else {
+            painterResource(id = R.drawable.gomeet_icon)
+          }
+      Row(
+          verticalAlignment = Alignment.CenterVertically,
+          modifier =
+              Modifier.padding(vertical = 10.dp)
+                  .background(color = backgroundColor, shape = RoundedCornerShape(10.dp))
+                  .clickable {
+                    nav.navigateToEventInfo(
+                        eventId = item.event.eventID,
+                        title = item.event.title,
+                        date = item.event.date.toString(),
+                        time = item.event.time.toString(),
+                        description = item.event.description,
+                        organizer = item.event.creator,
+                        loc = LatLng(item.event.location.latitude, item.event.location.longitude),
+                        rating = 0.0)
+                  }) {
+            Image(
+                painter = painter,
+                contentDescription = "Event Icon",
+                modifier = Modifier.size(64.dp).padding(8.dp))
+            Spacer(modifier = Modifier.width(16.dp))
+            Column {
+              Text(
+                  text = "${item.event.title}",
+                  modifier = Modifier.fillMaxWidth().padding(8.dp),
+                  color = Color.Black)
+              Text(
+                  text =
+                      "${item.event.date.format(DateTimeFormatter.ofPattern("dd-MM-yyyy"))} - ${item.event.time.format(DateTimeFormatter.ofPattern("HH:mm"))}",
+                  modifier = Modifier.fillMaxWidth())
+              Text("${item.event.description}", color = Color.Gray)
```

Comment:
Putting the whole description is risky. We don't have a limit to its length so a long description would make the UI look weird. I would suggest either removing the description completely or cutting it to a certain number of words